### PR TITLE
Add mender 2.1.1 and mender-artifact 3.2.0b1 recipes.

### PR DESCRIPTION
**Pull request by lluiscampos**

### CODE REVIEW

**Comment by lluiscampos** [comment]
Tested together with the release build at [Jenkins 4383](https://mender-jenkins.mender.io/job/mender-builder/4383/)

File: meta-mender-core/recipes-mender/mender/mender_2.1.1.bb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+#-------------------------------------------------------------------------------
+# THINGS TO CONSIDER FOR EACH RELEASE:
+# - SRC_URI (particularly "branch")
+# - SRCREV
+# - DEFAULT_PREFERENCE
+#-------------------------------------------------------------------------------
+
+SRC_URI = "git://github.com/mendersoftware/mender;protocol=https;branch=2.1.x"
+
+# Tag: 2.1.1
```
Comment (kacf) [inline meta-mender-core/recipes-mender/mender/mender_2.1.1.bb]: b1, right?

I know we don't usually do it for patch releases, but it's hard to change the pattern when the rest of the release is beta.

File: meta-mender-core/recipes-mender/mender/mender_2.1.1.bb [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+SRCREV = "53400c6cfd67a6ed28acb906ee2dfb896a4bf474"
+
+# Enable this in Betas, not in finals.
+# Downprioritize this recipe in version selections.
+#DEFAULT_PREFERENCE = "-1"
```
Comment (kacf) [inline meta-mender-core/recipes-mender/mender/mender_2.1.1.bb]: Following b1, this should be enabled, I think.

File: meta-mender-core/recipes-mender/mender/mender_2.1.1.bb (hunk 1 above)
Comment (lluiscampos) [reply meta-mender-core/recipes-mender/mender/mender_2.1.1.bb]: Discussed in person.

The tag for Mender client is 2.1.1 (not beta). Agreed in that it is inconsistent with a beta release, but it has been properly tested and the risk is very low.

**Comment by lluiscampos** [comment]
Added also mender-binary-delta v1.0.0b1 recipe

**Pull request merged**